syntax = "proto3";

package filetransfer;

service FileTransfer {
  rpc SendFile(FileRequest) returns (FileResponse);
}

message FileRequest {
  bytes content = 1;
}

message FileResponse {
  bool success = 1;
}